parsing/src/sheet.rs

```diff
@@ -0,0 +1,75 @@
+use std::{error::Error, fmt::Display, io::Cursor};
+
+use crate::{bits::PickBit, read::ReadExt};
+
+pub struct Sheet {
+    pub strokes: Vec<SheetStroke>,
+}
+
+#[derive(Debug)]
+pub struct SheetStroke {
+    pub x: u8,
+    pub y: u8,
+    pub draw_line: bool,
+    pub style_color: u8,
+    pub style_3d: bool,
+    pub style_bold: bool,
+}
+
+#[derive(Debug)]
+pub enum SheetStrokeDeserializeError {
+    InvalidFormat,
+}
+
+impl Display for SheetStrokeDeserializeError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        use SheetStrokeDeserializeError::*;
+        match self {
+            InvalidFormat => write!(f, "Invalid format"),
+        }
+    }
+}
+
+impl Error for SheetStrokeDeserializeError {}
+
+pub type SheetDataBytes = Vec<u8>;
```

Comment:
This is a pretty simple type so this isn't particularly necessary; the reason I did this in my other code was because that was a fixed-length array with a very particular size